media/js/mozorg/home/home.js

```diff
@@ -2,147 +2,171 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-(function(Waypoint) {
-    'use strict';
-
-    function getNextEl(el) {
-        el = el.nextSibling;
-        while (el) {
-            if (el.nodeType === 1) {
-                return el;
-            }
-            el = el.nextSibling;
-        }
-        return null;
-    }
-
-    /*
-     * Video Tracking
-     */
+ /* global YT */
+ /* eslint no-unused-vars: [2, { "varsIgnorePattern": "onYouTubeIframeAPIReady" }] */
+
+// YouTube API hook has to be in global scope
+ function onYouTubeIframeAPIReady() {
+     'use strict';
+ }
+
+ (function($, Waypoint) {
+     'use strict';
+
+     var tag = document.createElement('script');
+     tag.src = 'https://www.youtube.com/iframe_api';
+     var firstScriptTag = document.getElementsByTagName('script')[0];
+     firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+
+     function getNextEl(el) {
+         el = el.nextSibling;
+         while (el) {
+             if (el.nodeType === 1) {
+                 return el;
+             }
+             el = el.nextSibling;
+         }
+         return null;
+     }
+
+     function initVideoEvents() {
+         var videoCards = document.querySelectorAll('.mzp-c-card.has-video-embed .mzp-c-card-block-link');
+
+         function playVideo(event) {
+             var card = event.currentTarget;
+             var title = card.querySelector('.mzp-c-card-title').innerText;
+             var sibling = getNextEl(event.currentTarget);
+             var content = sibling.querySelector('.mzp-c-card-video-content');
+
+             var videoLink = document.querySelector('.video-play');
```

Comment:
This will always get the first element on the page that matches this selector, which works here but would break if we had multiple videos on the page. Should this be changed to `content.querySelector`?